examples/notebooks/healthcare/beam_post_hl7_messages_to_hcapi.ipynb

```diff
@@ -0,0 +1,484 @@
+        "# KIND, either express or implied. See the License for the\n",
+        "# specific language governing permissions and limitations\n",
+        "# under the License\n",
+        "\n",
+        "##################################\n",
+        "# Author: devanshmodi@goole.com  #\n",
+        "##################################\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [],
+      "metadata": {
+        "id": "RL1LDp645ogr"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# **Post Hl7v2 messages to HCAPI HL7v2 store pipeline**\n",
+        "\n",
+        "This example demonstrates how to set up an Apache Beam pipeline that reads a HL7 file from [Google Cloud Storage](https://https://cloud.google.com/storage), and calls the [Google Cloud Healthcare API Hl7v2 store to store Hl7 messages](https://cloud.google.com/healthcare-api/docs/how-tos/hl7v2-messages) to extract information from unstructured data. This application can be used in contexts such as reading raw Hl7 messages, if needed parse them or modify them as per your defined Hl7v2 store configurations and store data into Hl7v2 store.\n",
+        "\n",
+        "An Apache Beam pipeline is a pipeline that reads input data, transforms that data, and writes output data. It consists of PTransforms and PCollections. A PCollection represents a distributed data set that your Beam pipeline operates on. A PTransform represents a data processing operation, or a step, in your pipeline. It takes one or more PCollections as input, performs a processing function that you provide on the elements of that PCollection, and produces zero or more output PCollection objects.\n",
+        "\n",
+        "For details about Apache Beam pipelines, including PTransforms and PCollections, visit the [Beam Programming Guide](https://beam.apache.org/documentation/programming-guide/).\n",
+        "\n",
+        "You'll be able to use this notebook to explore the data in each PCollection."
+      ],
+      "metadata": {
+        "id": "wC9KRrlORwKu"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Lets install necessary packages"
+      ],
+      "metadata": {
+        "id": "81wCK9XnS6Sc"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install apache-beam[gcp]"
+      ],
+      "metadata": {
+        "id": "Yv1phmRZS23c"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**GCP Setup**\n",
+        "1. Authenticate your notebook by `gcloud auth application-default login` in the Colab terminal.\n",
+        "\n",
+        "2. Run `gcloud config set project <YOUR-PROJECT>`\n",
+        "\n",
+        "Set the variables in the next cell based upon your project and preferences. The files referred to in this notebook nlpsample*.csv are in the format with one\n",
```

Comment:
L115-118 are irrelevant to this notebook, right?

Reply:
Yes But they want to set the project they can